innovation_sweet_spots/getters/inputs.py

```diff
@@ -86,7 +87,32 @@ def get_cb_data(fpath=INPUTS_PATH, cb_data_spec_path=CB_DATA_SPEC_PATH):
     return tables
 
 
+def get_hansard_data(fpath=f"{INPUTS_PATH}hansard/"):
```

Comment:
HANSARD_PATH = INPUTS_PATH / "hansard"

```suggestion
def get_hansard_data(fpath=HANSARD_PATH):
```